src/app/panels/text-panel/text-panel.component.ts

```diff
@@ -27,8 +27,21 @@ export class TextPanelComponent {
     return this._mc;
   }
 
+  public selLayer: string;
```

Comment:
You can declare private like _mc on row 18

Reply:
Declaring it private generates error "src/app/panels/text-panel/text-panel.component.html:11:92 - error TS2341: Property 'selLayer' is private and only accessible within class 'TextPanelComponent'"

Reply:
sure, you have to change on HTML file and use selectedLayer because you have get selectedLayer declared on row 37